sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JacksonParser.scala

```diff
@@ -80,6 +80,9 @@ class JacksonParser(
     options.locale,
     legacyFormat = FAST_DATE_FORMAT,
     isParsing = true)
+  private lazy val timeFormatter = TimeFormatter(
+    options.timeFormatInRead,
+    isParsing = true)
```

Comment:
nit. This could be one-liner.

```scala
-  private lazy val timeFormatter = TimeFormatter(
-    options.timeFormatInRead,
-    isParsing = true)
+  private lazy val timeFormatter = TimeFormatter(options.timeFormatInRead, isParsing = true)
```

Reply:
Done